Alkiiro/refactor-tractor
# Accessibility - Up to 100 (Still need to do manual tests)

#### What's this PR do?
This PR updates the html document with alt tags, the html document with a language tag, and updates the base.scss with an adjusted color scheme for the text that was not accessible due to color contrast. 
#### Where should the reviewer start?
I would take a look at the base.scss and the index.html (both under the src folders). These are the only two files that were changed.
#### How should this be manually tested?
I would pull the repo down and then do an accessibility audit on our your machine and server. Want to make sure that this was implemented correctly and works on your computers!
#### Any background context you want to provide?
We will need to continue to check accessibility (especially tabbing as there is no tabIndex at all right now), but for now the audit comes back well. We should keep checking this and also do the manual tests.
#### What are the relevant tickets?
This should fix #2, but we still have a few accessibility issues. 
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/47511765/67251909-f4d7cd80-f42d-11e9-8c6a-a8e28dc3a9fc.png)
